lib/elixir/test/erlang/control_test.erl

```diff
@@ -128,6 +128,15 @@ not_test() ->
   {true, _} = eval("not false"),
   ?assertError(badarg, eval("not 1")).
 
+rearrange_not_left_in_right_test() ->
+  %% TODO: Deprecate `not left in right` rearrangement
```

Comment:
It'd better to replace backticks with regular double quotes, and we usually have period in comments if they are not titles.